src/action_types/channels.js

```diff
@@ -75,4 +75,8 @@ export default keyMirror({
     RECEIVED_CHANNEL_MEMBERS_MINUS_GROUP_MEMBERS: null,
 
     RECEIVED_TOTAL_CHANNEL_COUNT: null,
+
+    POST_UNREAD_FAILURE: null,
```

Comment:
Do we need to track the request?

Reply:
Not for functionality, it was suggested by @jespino and it might be useful in the future, so I added it as it didn't hurt. Let me know if you prefer to remove it

Reply:
@jespino any reason you wanted to track the request? If there is no clear reason we should definitely remove. “It does not hurt” is clearly an understatement cause by tracking the request you’ll cause the state to change at least 3 times which is not ideal

Reply:
you are right about the state. I'll remove it unless @jespino has a reason for that

Reply:
No, I have no reason, only consistency, but if we going to remove that, I would add a Jira ticket for removing unused request tracks there, because looks like everything is tracked, if we only have a small bunch of them is more obvious that is only used when is needed.

Reply:
I believe the majority were already removed but @sudheerDev can confirm

Reply:
@enahum Nope not all are removed. Only 2/5 were removed which effect center channel perf. 

@jespino @Willyfrog Lets not add them if we are not going to use them as we want to remove any of these flags which are not needed.

We cannot move away from this pattern completely as we might need to have the state if two components are looking for the status of these requests

Reply:
already removed!

Reply:
@enahum Correction more than half of the important request flags are removed. https://mattermost.atlassian.net/browse/MM-13544. 

But other two tickets which should have been on backlog got closed https://mattermost.atlassian.net/browse/MM-13545, https://mattermost.atlassian.net/browse/MM-13545. I just reopened them again and triaged. We can consider them for HW